lms/static/sass/shared-v2/_variables.scss

```diff
@@ -1,7 +1,7 @@
 // LMS variables
 
 $lms-gray: palette(grayscale, base);
-$lms-background-color: palette(grayscale, x-back);  // Why is the Pattern Library default background different?
+$lms-background-color: rgb(252, 252, 252);  // Correct shade of grey not available in the Pattern Library
```

Comment:
@clrux FYI we were unable to find a Pattern Library grey to use for the LMS background color. Is this the additional grey that you mentioned you were considering adding?